CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.2.2-dev
```

Comment:
Any time we are adding new API surface area, including new arguments, it should be a feature version bump.

```suggestion
## 2.3.0-dev
```